test/test_execution.py

```diff
@@ -227,6 +229,6 @@ def runner() -> int:
 
 
 def test_duct_as_executable(temp_output_dir: str) -> None:
-    ps_command = f"./src/con_duct/__main__.py -p {temp_output_dir} sleep 0.01"
+    ps_command = f"{sys.executable} {__main__.__file__} -p {temp_output_dir} sleep 0.01"
```

Comment:
Isn't this the equivalent of 'python thefile.py'?

This makes sense to use the same version of python, but IIUC it doesn't prove the direct executability of the file.

Reply:
> Isn't this the equivalent of 'python thefile.py'?

yes

> This makes sense to use the same version of python, but IIUC it doesn't prove the direct executability of the file.

you are correct if by "executability" you mean an executable permissions/use of shebang.  The problem is that AFAIK that executable permission does not get copied while building/deploying `con-duct` so I had to specify the interpreter (`sys.executable`) directly to get it running.  The main point of the test though IMHO to verify that the script is "self sufficient" and does not rely on imports (ideally we should even copy it outside somewhere and run from there... later)